application/models/Keyword_model.php

```diff
@@ -9,6 +9,19 @@ public function get_applied($list_keywords)
 
 
     }
+    
+    	public function autocomplete($term)
+	{
+		$sql = 'SELECT DISTINCT k.value 
+		FROM keywords k 
+		JOIN project_keywords pk
+		ON k.id = pk.keyword_id
+		WHERE k.value like "' . $term . '%" AND pk.project_id IS NOT NULL
```

Comment:
Agreed. As you can see, I'm no hotshot developer! (Much more a Librivox reader than a Librivox techie...) 

It may take me some little time as I have a few social commitments, but I expect I'll be updating my pull request to incorporate most of the changes you've suggested within the next 24 hours.

Reply:
> I'm no hotshot developer!

Nah, you've done well! The auto-complete stuff is tricky to put together and you got there. The main thing is that you've made it happen, and I'm looking forward to using it.

> Much more a Librivox reader

Well, you're a long way ahead of my score of 2 sections that's for sure :wink: